### DISCUSSION TITLE
Multiple Selection In Inspector

### DISCUSSION THREAD

**Discussion by Fractal393** [clientIO/joint#2532] [Q&A] [answered]
### Introduction

I have used the multiple selection as present in the documentation. But in order to select multiple options I have to hold down ctrl and select them. Is there a way where i can use checkboxes so it's easy for the users?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
You can render a checkbox next to an element as shown in this example: https://codepen.io/jointjs/pen/abQyLew

**Reply by Fractal393** [reply]
I want to use it for the inspector not the stencil. Basically every element has a property called list. Now this list variable is shown in the inspector with respect to a dropdown. This dropdown is a multi-select where I have to hold down Ctrl to do it. I need checkboxes instead is it possible?

**Reply by kumilingus** [reply, marked as answer]
Ok, sorry, I didn't read your question properly 🤦 

You can use `select-button-group`:
```
workdays: {
    type: 'select-button-group',
    options: ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'],
    multi: true,
}
```
Or define a custom field: https://resources.jointjs.com/docs/rappid/v4.0/ui.Inspector.html#ui.Inspector.custom-fields

**Reply by Fractal393** [reply]
Thank You!
